components/automatic_points_table/commons/automatic_points_table.lua

```diff
@@ -0,0 +1,106 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:AutomaticPointsTable
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+---- This module creates a Points Table that automatically gets data from the prizepools
+---- of set tournaments
+
+local Class = require('Module:Class')
+-- local Table = require('Module:Table')
+
+local DivTable = require('Module:DivTable')
+local Json = require('Module:Json')
+local split = require('Module:StringUtils').split

```

Comment:
I was told we prefer (code style wise) it to import the whole module and add the function index call at the line where it is used
```suggestion
local String = require('Module:StringUtils')

```